src/services/requiredStandardsService.ts

```diff
@@ -58,11 +58,11 @@ export class RequiredStandardsService {
     results: ITaxonomySectionRequiredStandards[],
     euVehicleCategory: string,
   ): DefectGETRequiredStandards {
+    const categoryEnumKey = getEnumKeyByValue(EUVehicleCategory, euVehicleCategory);
```

Comment:
Rather than adding logic into the defects service to handle the typo's, wouldn't it be easier to fix the enum values in the types definition package?

Otherwise any other services that use enum in the types definition would also need to apply logic in their code to resolve.

Reply:
I've already tried correcting the typo's manually, inside of the node modules and that didn't resolve it. And using a hyphen in the enum key makes it throw an error, saying that you're not allowed to do that

Reply:
Yes I see your point about the -, forgot enum keys had to be a string literal/number. 

We should still be able to update it for l1e and l2e though. If it's not done as part of this ticket we just need to make sure we update those 2 the next time any changes are made to the types definition package.

Reply:
would you be happy with a mix of both, I'll correct the type definition enum and keep the function there to handle l1e-a?

Reply:
Yes absolutely, if you are good to do that. Thanks.